CMakeLists.txt

```
@@ -256,6 +256,16 @@ if(    yaml-cpp_FOUND
     INTERFACE_INCLUDE_DIRECTORIES "${YAML_CPP_INCLUDE_DIR}"
   )
 endif()
+# Workaround to fix a problem with the imported target name
+# The target name includes the namespace yaml-cpp begining
+# from version 0.8.0. So the new exported target is now
+# yaml-cpp::yaml-cpp.
+# To work also with previous versions of yaml-cpp an alias
+# is created and the new target name is used in the project
+if(yaml-cpp_FOUND AND TARGET yaml-cpp)
```

Would this be too paranoid?

```suggestion
if(yaml-cpp_FOUND AND TARGET yaml-cpp AND NOT TARGET yaml-cpp::yaml-cpp)
```

---

I don't know if this is too paranoid but it shouldn't hurt. I will try if I see problems or if it simply works.